### ISSUE TITLE
Can't get out of pass through mode on facebook

### ISSUE THREAD

**Issue by anoda9** [vimperator/vimperator-labs#273]
If you're on the facebook newsfeed and activate pass through mode, pressing shift+escape a second time has no effect and won't toggle out of it.

**Comment by gkatsev** [comment]
Yeah, it's an issue, probably related to weird javascript keyboard handling. `<Insert>` seems to end up working fine for escaping.

**Comment by ghallberg** [comment]
I've had similar issues on many pages, for example https://paper.dropbox.com
